### ISSUE TITLE
[BUG] Cosmos Bulk Insert: IllegalAccessError: com/azure/cosmos/ClientSideRequestStatistics$SystemInformation

### ISSUE THREAD

**Issue by jeffpeiyt** [Azure/azure-sdk-for-java#18834]
**Describe the bug**

When executing a bulk upsert code 95% similar to https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosBulkTest.java#L89 on `upsertItem_withbulk `

The operation was successful but this warning message is annoying; showed ~8 times when I inserted 20 documents. 

Looks similar to https://github.com/Azure/azure-sdk-for-java/issues/11589 and https://stackoverflow.com/questions/46080924/jackson-afterburnermodule-gives-a-warning-in-the-log but on a different property. 

***Exception or Stack Trace***
```
2021-01-27 10:05:33 WARNING [com.fasterxml.jackson.module.afterburner.ser.OptimizedBeanPropertyWriter] Disabling Afterburner serialization for class com.azure.cosmos.ClientSideRequestStatistics$SystemInformation (field #2; mutator com.fasterxml.jackson.module.afterburner.ser.StringMethodPropertyWriter), due to access error (type java.lang.IllegalAccessError, message=com/azure/cosmos/ClientSideRequestStatistics$SystemInformation)
 java.lang.IllegalAccessError: com/azure/cosmos/ClientSideRequestStatistics$SystemInformation
	at com.azure.cosmos.ClientSideRequestStatistics$SystemInformation$Access4JacksonSerializerbf8f03f4.stringGetter(com/azure/cosmos/ClientSideRequestStatistics$SystemInformation$Access4JacksonSerializer.java)
	at com.fasterxml.jackson.module.afterburner.ser.StringMethodPropertyWriter.serializeAsField(StringMethodPropertyWriter.java:44)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:719)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:155)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValue(ObjectMapper.java:2656)
	at com.fasterxml.jackson.core.base.GeneratorBase.writeObject(GeneratorBase.java:381)
	at com.fasterxml.jackson.core.JsonGenerator.writeObjectField(JsonGenerator.java:1726)
	at com.azure.cosmos.ClientSideRequestStatistics$ClientSideRequestStatisticsSerializer.serialize(ClientSideRequestStatistics.java:323)
	at com.azure.cosmos.ClientSideRequestStatistics$ClientSideRequestStatisticsSerializer.serialize(ClientSideRequestStatistics.java:272)
	at com.fasterxml.jackson.databind.SerializerProvider.defaultSerializeValue(SerializerProvider.java:1033)
	at com.fasterxml.jackson.databind.node.POJONode.serialize(POJONode.java:114)
	at com.fasterxml.jackson.databind.node.ObjectNode.serialize(ObjectNode.java:311)
	at com.fasterxml.jackson.databind.ser.std.SerializableSerializer.serialize(SerializableSerializer.java:39)
	at com.fasterxml.jackson.databind.ser.std.SerializableSerializer.serialize(SerializableSerializer.java:20)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
	at com.fasterxml.jackson.databind.ObjectMapper._configAndWriteValue(ObjectMapper.java:3906)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3220)
	at com.azure.cosmos.CosmosException.toString(CosmosException.java:342)
	at java.lang.Throwable.<init>(Throwable.java:311)
	at java.lang.Exception.<init>(Exception.java:102)
	at java.lang.RuntimeException.<init>(RuntimeException.java:96)
	at java.util.concurrent.CompletionException.<init>(CompletionException.java:88)
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:769)
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:736)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at com.azure.cosmos.implementation.directconnectivity.rntbd.RntbdRequestManager.messageReceived(RntbdRequestManager.java:856)
	at com.azure.cosmos.implementation.directconnectivity.rntbd.RntbdRequestManager.channelRead(RntbdRequestManager.java:181)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:328)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:302)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:438)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:255)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:287)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1475)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1224)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1271)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:505)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:444)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:283)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1422)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:931)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:700)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:635)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:552)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:514)
	at io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1044)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

**To Reproduce**

My code is 95% similar to https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosBulkTest.java#L89 on `upsertItem_withbulk `

***Code Snippet***
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosBulkTest.java#L89 on `upsertItem_withbulk `

**Expected behavior**

Should have no warning message or stack trace

**Screenshots**

**Setup (please complete the following information):**
 - OS: mac OS 11.1
 - IDE : IntelliJ
 - Version of the Library used:  "com.azure:azure-cosmos:4.11.0"

**Additional context**

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by kushagraThapar** [comment]
@moderakh @simplynaveen20  - another issue on the jackson after burner logs. 
@jeffpeiyt - thanks for filing this issue, we will get back to you shortly.

**Comment by jeffpeiyt** [comment]
@kushagraThapar thanks for the prompt attention!

**Comment by meganalnico** [comment]
Any workaround for this? My CosmosDB POC project is dead in the water unless I can get around this.

**Comment by kushagraThapar** [comment]
@meganalnico and @jeffpeiyt  - we did some improvements on this in the latest version of the `azure-cosmos` SDK - https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-sdk-java-v4#4120-2021-02-09
Can you please try out that version ?

**Comment by meganalnico** [comment]
Sorry, that's the version I'm using. 

`implementation group: 'com.azure', name: 'azure-cosmos', version: '4.12.0'`

**Comment by kushagraThapar** [comment]
Good to know that, thanks for the version information. 
@moderakh  @simplynaveen20 - this is a blocking issue, can you please prioritize this ?

**Comment by meganalnico** [comment]
Looks like these exceptions were hiding other problems. I seem to be up and running.

**Comment by kushagraThapar** [comment]
@jeffpeiyt - this issue has been fixed in v4.14.0 . Please test with the latest version and re-open the issue if it still exists.
